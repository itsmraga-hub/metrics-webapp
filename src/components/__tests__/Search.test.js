import renderer from 'react-test-renderer';

import SearchBar from '../Search';

it('<SearchBar /> Renders Properly', () => {
  const tree = renderer.create(
    <SearchBar />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
