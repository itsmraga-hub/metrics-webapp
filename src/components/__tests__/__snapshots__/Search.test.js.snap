// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> Renders Properly 1`] = `
<div
  className="SearchContainer"
>
  <input
    className="InputField"
    id="searchbar"
    name="search"
    placeholder="Filter/search by country"
  />
</div>
`;
